app/controllers/authenticate_controller.rb

```diff
@@ -70,7 +70,15 @@ def authenticate(input = authenticator_input)
       authenticators: installed_authenticators,
       enabled_authenticators: Authentication::InstalledAuthenticators.enabled_authenticators_str(ENV)
     )
-    render json: authn_token
+    content_type = :json
+    if encoded_response?
+      # The response will be a base64 encoded version of the JWT.
+      Rails.logger.debug("Responding with base64 encoded JWT")
+      content_type = :plain
+      authn_token = ::Base64.encode64(authn_token.to_json).delete("\n").strip
```

Comment:
extract to method?

Reply:
What do you think on https://apidock.com/rails/String/squish ? is it solving the issue?

Reply:
the `to_json` is mendetory?

Reply:
> What do you think on https://apidock.com/rails/String/squish ? is it solving the issue?

Cool! we need to add `require 'active_support/core_ext/string/filters'`. Is that OK? @uCatu

Reply:
> the `to_json` is mendetory?

Yes.  We encode the json format.